pkg/session/bootstrap.go

```diff
@@ -62,6 +63,8 @@ import (
 	"go.uber.org/zap"
 )
 
+var bootstrapOwnerKey = "/tidb/distributeLock/"
```

Comment:
```suggestion
var bootstrapOwnerKey = "/tidb/distributeDDLOwnerLock/"
```
Should this be more clear about the usage of this distribute lock.

Reply:
Please also add comment for this variable‘s usage.